docs/site/_april_milestone.md

```diff
@@ -0,0 +1,77 @@
+# April Milestone
+
+## Overview
+
+With COVID-19 impacting the maintainers (like everyone else in the world!) at
+different levels, we will be committing less story points to give us more
+flexibility.
+
+The goals for this month would be to:
+
+- Aim at finish the
+  [Migration guide epic](https://github.com/strongloop/loopback-next/issues/3935)
+  except for the
+  [migrate LB3 components related tasks](https://github.com/strongloop/loopback-next/issues/3955).
+- Finish
+  [From model definition to REST API with no custom repository/controller classes epic](https://github.com/strongloop/loopback-next/issues/3935)
+
+## Goals
+
+- [ ] Migration guide #3935
+
+  - [ ] :running: Migration Guide: Discuss life cycle differences between LB3
+        and LB4 #3935
+  - [ ] :running: How to migrate apps using loopback-component-passport #3958
+  - [ ] Migration Guide: Boot up #4838
+  - [ ] Migration Guide: Custom validation #4837
+  - [ ] Spike: How to migrate LB3 components #4099
+  - [ ] How to map LB3 CLI commands to LB4 CLI #3953
+  - [ ] Spike: How to migrate apps using loopback-component-oauth2 #3959
+  - [ ] How to run LB3 app tests when the app is mounted in a LB4 project #3978
```

Comment:
@deepakrkris, please replace #3978 with:
Spike: how to run LB3 tests from LB4 when LB3 is mounted on the LB4 app #5004

since we've created a spike on it. thanks.